catalog/imaging/imaging-transformer-nitf/src/test/java/org/codice/alliance/transformer/nitf/TestCommonTreAttributes.java

```diff
@@ -0,0 +1,23 @@
+package org.codice.alliance.transformer.nitf;
+
+import static org.hamcrest.core.IsNull.notNullValue;
+import static org.junit.Assert.assertThat;
+
+import java.util.stream.Stream;
+
+import org.codice.alliance.transformer.nitf.common.AcftbAttribute;
+import org.codice.alliance.transformer.nitf.common.AimidbAttribute;
+import org.codice.imaging.nitf.core.common.NitfFormatException;
+import org.junit.Test;
+
+public class TestCommonTreAttributes {
```

Comment:
According to testing guidelines, this test class should be named `CommonTreAttributesTest.java`